data/types/common/day.go

```diff
@@ -0,0 +1,23 @@
+package common
+
+const (
+	DaySunday    = "sunday"
```

Comment:
No, Uploader uploads sleep schedules with days of the week capitalized. 

Are Dexcom alert schedules currently being uploaded from anywhere? I don't think we have a ticket for Uploader yet.

Reply:
Yes, Dexcom alert schedules are currently uploaded via the Dexcom Account Connection (and have been for years).

When you say "Uploader uploads sleep schedules with days of the week capitalized", do you mean to Jellyfish?

It'd be great if we could get this consistent (specifically, all use lowercase), but if that's going to be too much trouble...

Reply:
> When you say "Uploader uploads sleep schedules with days of the week capitalized", do you mean to Jellyfish?
Yes.

I guess this could be one more thing we add to the migration? If we're changing `bolus` to `boluses`, we could also changed `Monday` to `monday`. It's an easy change in Uploader.

Reply:
Yes, I suppose we should do this. One more thing to add to the list. @jh-bate 

I wouldn't be surprised if we find other issues once you (@gniezen) actually start uploading via Platform. Considering the lack of validation in Jellyfish (and the validation in Platform), there will probably be other discrepancies that we'll have to resolve.